.github/workflows/release-production.yml

```diff
@@ -0,0 +1,42 @@
+name: Release Production Environment
+
+on:
+  push:
+    branches: [ v* ]
```

Comment:
This should be for tags, not branch, right?

Reply:
Fixed